File: 2037-minimum-number-of-moves-to-seat-everyone/2037-minimum-number-of-moves-to-seat-everyone.py
class Solution(object):
    def minMovesToSeat(self, seats, students):
        """
        :type seats: List[int]
        :type students: List[int]
        :rtype: int
        """
        seats.sort()
        students.sort()
        n = len(seats)
        res = 0
        for i in range(n):
            res+=abs(students[i] - seats[i])
        return res

File: 2037-minimum-number-of-moves-to-seat-everyone/2037-minimum-number-of-moves-to-seat-everyone.go
func AbsoluteValue(x int) int {
    if x < 0 {
        return -x
    }
    return x
}
func minMovesToSeat(seats []int, students []int) int {
    sort.Ints(seats)
    sort.Ints(students)
    n := len(students)
    res := 0
    for i:=0; i < n; i++ {
        res += AbsoluteValue(students[i]-seats[i])
    }

    return res
    
}